contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -346,6 +348,35 @@ def _execute(self,
         self._verify_existence_of_tracks()
 
 
+class CaptureRepeatedly(E2ETestCase):
+    """
+    This test case simulates a user quickly starting and stopping captures by "smashing" the F5 key. It does not verify
+    any behavior, but it will fail if Orbit crashes.
+    """
+
+    def _stop_capture_if_necessary(self):
+        logging.info('Querying if a capture is still running')
+        capture_tab = self.find_control('Group', "CaptureTab")
+        capture_options_button = self.find_control('Button', 'Capture Options', parent=capture_tab)
+        if not capture_options_button.is_enabled():
```

Comment:
Should that trigger an error somehow? In this test F5 is hit an even number of times - so a running capture would indicate an issue, right?

Reply:
F5 is pressed many more times than captures are started/stopped, because starting and stopping a capture takes time (communication, finalization, etc...). This test is indeed simulating "smashing" the key, it's not waiting for the capture to be started or stopped, otherwise the delays would water it down.